style.css

```diff
@@ -1,98 +1,280 @@
+    font-family: var(--secondary-font-family);
     font-size: 24px;
+    font-style: normal;
     font-weight: 700;
     line-height: 32px;
-    font-family: var(--secondary-font-family);
   }
 
-  p {
-    color: var(--secondary-color);
+  .about-me p {
+    color: var(--secondary-font-color);
     font-family: var(--default-font-family);
     font-size: 16px;
+    font-style: normal;
     font-weight: 400;
     line-height: 24px;
-    margin-top: 8px;
   }
 
-  nav {
-    height: 48px;
-    padding: 2px 24px 2px 12px;
+  .social-links {
+    padding: 8px;
   }
 
-  nav ul {
-    margin: 0;
-    padding: 0;
+  .social-links a {
+    color: #dbd8d7;
+    width: 35px;
+    height: 35px;
+    padding: 2px 3px;
+    border-radius: 8px;
   }
 
-  nav li {
+  .social-links a:hover {
+    color: var(--default-font-color);
+  }
+
+  .grid-container {
+    display: grid;
+    padding: 120px 0;
+    align-items: center;
+    gap: 80px;
+  }
+
+  header {
+    color: var(--default-background-color);
+    font-family: var(--primary-font-family);
+    font-size: 40px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 52px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+  }
+
+  .feature {
+    display: flex;
+    padding: 0 24px;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+  }
+
+  .img {
+    width: 327px;
+    height: 237px;
+  }
+
+  h4 {
+    color: #091e42;
+    font-family: var(--primary-font-family);
+    font-size: 32px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 44px;
+  }
+
+  .tags ul {
+    display: flex;
+    align-items: center;
+    gap: 12px;
+  }
+
+  .tags li {
+    color: #091e42;
+    font-family: var(--third-font-family);
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
     list-style: none;
-    line-height: 42px;
-    margin-left: 15px;
+    border: 1px solid #8993a4;
+    background: var(--secondary-font-color);
+    display: flex;
+    padding: 10px 12px;
+    align-items: center;
   }
 
-  .navbar-brand {
-    color: var(--secondary-color);
+  button {
+    display: flex;
+    padding: 12px;
+    border: none;
+    align-items: center;
+    gap: 10px;
+    background: var(--default-font-color);
+    color: var(--secondary-font-color);
     text-align: center;
-    font-family: var(--secondary-font-family);
-    font-size: 18px;
+    font-family: var(--third-font-family);
+    font-size: 17px;
     font-style: normal;
     font-weight: 700;
     line-height: 24px;
+    letter-spacing: 0.51px;
   }
 
-  nav a {
-    text-decoration: none;
-    color: var(--primary-color);
-    transition: all 100ms ease-in-out;
+  button:hover {
+    cursor: pointer;
   }
 
-  nav a:hover {
-    color: var(--branding-color);
+  .card-wrapper {
+    display: flex;
+    padding: 0 24px;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
   }
```

Comment:
Kindly remove `padding: 0 24px;` from the card-wrapper div too to fix the above problem.